kitsune/motidings/templates/motidings/unsub.html

```diff
@@ -15,7 +15,7 @@ <h1 class="title">{{ _('Are you sure you want to unsubscribe?') }}</h1>
       {{ csrf() }}
       <div class="submit-or-cancel">
         <input type="submit" value="{{ _('Unsubscribe') }}" />
-        <a href="{{ url('home') }}"{# Discards l10n. Better ideas? #}>{{ _('Cancel') }}</a>
+        <a href="{{ url('home') }}">{{ _('Cancel') }}</a>
```

Comment:
I don't understand this comment. Do you? Was it just a left over from when this template was copied from tidings?